domain/user/model/User.py

```diff
@@ -0,0 +1,8 @@
+from uuid import UUID
+
+
+class User:
+    def __init__(self, _id: UUID, username: str, password: str):
+        self.id = _id
+        self.username = username
+        self.password = password
```

Comment:
So think about override _str method for logging:
```
    def __str__(self) -> str:
        _dict = self.__dict__
        _dict.pop("password")
        return f'User{_dict}'
```

Reply:
Unfortunatly, python didn't like us overriding superfunction. When I implemented this, User's password field stopped working.
![image](https://user-images.githubusercontent.com/43717934/180571839-e667aae3-e639-4e61-be5f-6dbcc91b13dd.png)

I implemented my function instead and it works:
![image](https://user-images.githubusercontent.com/43717934/180571970-61e8408f-e3f3-4f2b-9a44-58ae219eaced.png)

Reply:
I also found out, that we need to add to `self.__dict__`  `.copy()` or it will destroy entire User class